python/test/t_HSICEstimator_std.py

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env python
```

Comment:
Just out of curiosity, is it necessary to test the `HSICEstimator` interface class, which in theory should never be called by the user in itself, but only through the derived Global/Conditional/Target sensitivity classes? Especially since if I' not mistaken  `HSICEstimator` is in practice identical to `HSICEstimatorConditionalSensitivity`.

Reply:
You are right. My reasoning was simple: the class is written so I test it.